### ISSUE TITLE
Can't log in to the admin area

### ISSUE THREAD

**Issue by bobemoe** [Kunstmaan/KunstmaanBundlesStandardEdition#129]
When browsing to /admin/ I'm redirected to /login/ as normal, upon entering the correct admin user/pass and clicking login, I am redirected back to the homepage rather than being logged in and redirected to /admin/dashboard/ (debug bar confirms I'm not logged in) If I use incorrect user/pass the login form is redisplayed, but no "wrong user/pass" message is displayed like you normally see.

The strange thing is I maintain many different sites using the the same repo/branch, and they all work fine!  The only difference is that I am not using our usual server setup running apache, instead we are using nginx.

I have tried the AcmeDemoBundle's Secure Area demo, and I can log in there just fine, so this shows fresh Symfony and nginx config is working fine togerher.

Any ideas? Or know issues with nginx?

Thanks

**Comment by bobemoe** [comment]
OK, looks like its a filesystem permissions thing.  The project folder and web root are mounted as an NFS share as the machine is provisioned with vagrant.

This mount and all files in it are owned by a random user and group, however all the files/folders are actually writable. If I copy the project to a local folder everything starts to work just fine.

Symfony doesn't have any problems writing to the cache and log folders, any user can write to any folder in this NFS mount, and all our other web apps work fine with this set up, so I don't really understand why this is causing the described behaviour.  

If anyone is interested, this link describes in a bit more detail how the NFS is mounted: https://groups.google.com/forum/?fromgroups=#!topic/vagrant-up/-J3UEqYXveA

**Comment by krispypen** [comment]
this is strange, if you are redirected to the homepage, then maybe the user don't have any roles. also the acmedemobundle uses an in memory security provider.

**Comment by roderik** [comment]
For the random user problem you can install [vagrant-bindfs](https://github.com/gael-ian/vagrant-bindfs). Recently learned about it from puphpet.com

> If you are on OSX or Linux, it is highly recommended you choose "NFS" as the Shared Folder Type, and install the vagrant-bindfs plugin with $ vagrant plugin install vagrant-bindfs.

Did you by any chance go for the single language setup? I noticed similar behaviour because in the config there is a new caching setting that needs to be changed for single language setups. (and is for now undocumented)

https://github.com/Kunstmaan/KunstmaanBundlesStandardEdition/blob/master/app/config/config_prod.yml#L37

**Comment by bobemoe** [comment]
Hi, thanks for the replies, I had looked briefly into the bindfs, but the setup guide I looked at didn't look that straightforward.  I'm actually on Linux rather than MacOS but I believe bindfs works on both.  Maybe I'll investergate a little futher.

Yes, I am using the single lang site.  I'm actually on Kunstmaan 2.x ranther than the new 3.x branch so I dont think that config change applies to me? Didnt realise 3.x had been released, yay \o/  I'm havign a read and looking into a migration plan but I'm finding 3.x a little unstable atm.
